pandas/tests/tseries/offsets/test_fiscal.py

```diff
@@ -625,3 +614,20 @@ def test_bunched_yearends():
     assert fy.rollback(dt) == Timestamp('2002-12-28')
     assert (-fy).apply(dt) == Timestamp('2002-12-28')
     assert dt - fy == Timestamp('2002-12-28')
+
+
+def test_fy5253_last_onoffset():
+    offset = FY5253(n=-5, startingMonth=5, variation="last", weekday=0)
```

Comment:
add a comment to these tests about what you are testing. next person to come along will have no idea. add the issue number as well.